templates/UmbracoProject/UmbracoProject.csproj

```diff
@@ -26,15 +26,6 @@
     <RazorCompileOnBuild>false</RazorCompileOnBuild>
     <RazorCompileOnPublish>false</RazorCompileOnPublish>
   </PropertyGroup>
-
-  <ItemGroup>
-    <Content Update="appsettings.*.json">
-      <DependentUpon>appsettings.json</DependentUpon>
-    </Content>
-    <Content Update="appsettings-schema.*.json">
-      <DependentUpon>appsettings-schema.json</DependentUpon>
-    </Content>
-  </ItemGroup>
```

Comment:
This has been moved to the `Umbraco.Cms.Targets.targets` file that is automatically imported, cleaning up the project file and also ensuring this will always be applied (in case you upgrade a v9/v10 site that doesn't include this).